File: backend/src/main/java/com/iit/admin/dto/AttendanceDTO.java
package com.iit.admin.dto;

import lombok.Data;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
public class AttendanceDTO {
    private Long id;
    private Long userId;
    private String username;
    private String userEmail;
    private String roleName;
    private String name;
    private LocalDate attendanceDate;
    private LocalDateTime punchIn;
    private LocalDateTime punchOut;
    private String status;
    private String source;
    private Double latitude;
    private Double longitude;
    private String cardUid;
}

File: android/app/src/main/java/com/iit/cliqchat/data/remote/dto/AttendanceDto.kt
package com.iit.cliqchat.data.remote.dto

data class AttendanceDto(
    val id: Long?,
    val userId: Long,
    val username: String,
    val userEmail: String,
    val roleName: String,
    val name: String?,
    val attendanceDate: String,
    val punchIn: String?,
    val punchOut: String?,
    val status: String,
    val source: String,
    val latitude: Double?,
    val longitude: Double?,
    val cardUid: String?
)
